### PR TITLE
docs(DocsLayout): fix max-width

### PR DESCRIPTION
**Pull request by mainframev**
fixed the container width for DocLayout, it has to be the same as navigation bar

### CODE REVIEW

File: docs/src/components/DocLayout/index.tsx [hunk 1]
```diff
@@ -113,7 +113,9 @@ export default function DocLayout({
               display: flex;
               margin: 0 auto;
               width: 100%;
-              max-width: 80rem;
+              box-sizing: border-box;
+              padding: 0 2rem;
+              max-width: 90rem;
```
Comment (DSil) [inline docs/src/components/DocLayout/index.tsx:118]: We can use the `MAX_CONTENT_WIDTH` global const, no?

File: docs/src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -44,7 +44,6 @@ const StyledWrapper = styled.header`
 const StyledInner = styled.div`
   max-width: ${MAX_CONTENT_WIDTH};
   padding: 0 ${CONTENT_PADDING};
-  box-sizing: content-box;
```
Comment (DSil) [inline docs/src/components/Navbar/index.tsx:47]: If we remove this, I also don't think it gets according to the designs 🤔 it gets smaller than the content in the homepage

File: docs/src/components/DocLayout/index.tsx (hunk 1 above)
Comment (mainframev) [reply docs/src/components/DocLayout/index.tsx:118]: yes

File: docs/src/components/Navbar/index.tsx (hunk 2 above)
Comment (mainframev) [reply docs/src/components/Navbar/index.tsx:47]: now it's correct, in Figma it's 1440px including paddings, before it was bigger.

Comment (mainframev) [reply docs/src/components/Navbar/index.tsx:47]: changed the size of HeaderBox, so it's visually ok

**Review by DSil — APPROVED** [review]

**Pull request merged**